src/main/java/Duke.java

```diff
@@ -1,10 +1,184 @@
+import java.util.Scanner;
+import static java.lang.Integer.parseInt;
+
 public class Duke {
+    public static final String LINE = "____________________________________________________________\n";
     public static void main(String[] args) {
+        printHelloMessage();
+        Scanner input = new Scanner(System.in);
+        Task[] tasks = new Task[100];
+        int index = 0;
+        processUserInput(input, tasks, index);
+    }
+
+    /**
+     * Processes user input after the app starts
+     * @param input User input
+     * @param tasks Task list represented by an array of Tasks
+     * @param index Index number of the newest Task added into tasks array
+     */
+    private static void processUserInput(Scanner input, Task[] tasks, int index) {
+        String command = "";
+        while (!(command.equalsIgnoreCase("bye"))) {
+            command = input.nextLine();
+            if ((command.equalsIgnoreCase("bye"))) {
+                break;
+            }
+            if (command.equalsIgnoreCase("list")) {
+                printList(tasks, index);
+            } else if ((command.toLowerCase()).contains("done")) {
+                markItemAsDone(tasks, command);
+            } else if ((command.toLowerCase()).contains("todo")) {
+                addTodoTask(tasks, index, command);
+                printAddTaskMessage(index, tasks[index]);
+                index++;
+            } else if ((command.toLowerCase()).contains("deadline")) {
+                if (command.contains("/by")) {
+                    addDeadlineTask(tasks, index, command);
+                    printAddTaskMessage(index, tasks[index]);
+                    index++;
+                } else {
+                    System.out.println("Please include deadline: /by deadline");
+                }
+            } else if ((command.toLowerCase()).contains("event")) {
+                if (command.contains("/at")) {
+                    addEventTask(tasks, index, command);
+                    printAddTaskMessage(index, tasks[index]);
+                    index++;
+                } else {
+                    System.out.println("Please include event time! /at event time");
+                }
+            } else {
+                System.out.println("Please specify type (deadline, event, todo) of task before Task description");
+            }
```

Comment:
Any reason why you used if statements instead of switch statements?

Reply:
I wasn't sure if the code `command.toLowerCase()).contains("commandWord")` will be a long line of code to implement as a case in the switch statement but I shall give it a try. Thank you for your suggestion! 😃